crates/sui-indexer/migrations/pg/2024-09-25-135801_event_emit_module_pruning_index/down.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX CONCURRENTLY IF EXISTS event_emit_module_tx_sequence_number;
```

Comment:
any reason that we want to separate these extra indices instead of putting them in one file, which seems less verbose on file layout?

Reply:
> Note that because these indices are being added to existing tables, they are being added using CONCURRENTLY IF NOT EXISTS, and because of that, they need to each be in their own migration file (concurrent index creation cannot go in a DB transaction).

From the PR description.

Reply:
I missed that, thanks!